.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
You have got `.eslintrc`, but how about config for `prettier`?
Just few useful articles:
1. https://maxpfrontend.ru/vebinary/nastroyka-eslint-prettier-pre-commit-hook-create-react-app-visual-studio-code/
2. https://ru.hexlet.io/blog/posts/rukovodstvo-eslint-prettier

Also, please, pay attention to the `pre-commit hook` in the first article. This is a very useful control mechanism while developing

Reply:
I think in this case .prettierrc doesn't matter because main rules determine in linter.
pre-commit hook - I will look at it.